docs/fidelity-bonds.md

```diff
@@ -74,33 +109,37 @@ fidelity bonds in it.
 
 This example creates an address which locks any coins sent to it until April 2020.
 
-    (jmvenv) $ python3 wallet-tool.py testfidelity.jmdat gettimelockaddress 2020-4
+    (jmvenv) $ python3 wallet-tool.py testfidelity.jmdat gettimelockaddress 2025-1
```

Comment:
You changed 2020-4 to 2025-1 here, but forgot to change "April 2020" to "January 2025" above.